src/team_manager/managers.py

```diff
@@ -0,0 +1,31 @@
+from django.utils.translation import gettext_lazy as _
+from django.contrib.auth.base_user import BaseUserManager
+
```

Comment:
What is the purpose of this file?

Reply:
Since in this project, we decided to take email as a username (we no longer have a username in this project), I used this to customize the user (both for user or superuser)